app/jobs/weekly_reminder.rb

```diff
@@ -7,7 +7,7 @@ class WeeklyReminder
 
   def perform
     User.find_each do |user|
-      if user.email != 'admin.test.com'
+      if user.email != 'admin@test.com'
```

Comment:
This email can be configurable in a ENV var as well

Reply:
you're right